database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -21,10 +21,10 @@ public function up()
             $table->string('password');
             $table->string('phone_number');
             $table->date('dob');
-            $table->string('profile_img');
+            $table->string('profile_img')->nullable();
```

Comment:
Please add migration for each change on the schema, since you change here will not work with your teamates unless they fresh the db.